index.js

```diff
@@ -44,13 +44,56 @@ function reduce(array) {}
 
 function reduceRight(array) {}
 
-function reverse(array) {}
+function reverse(array) {
+  const newArray = []; //wrote this as array(), getting confused with PHP;
+  for (let i = (array.length-1); i >= 0; i--){ //created infinite loop and issue with i being > than 0 rather than <: for (let i = (array.length-1); i <= array.length; i--){
+    newArray.push(array[i]);
```

Comment:
You are using native array method here (`newArray.push(array[i]);`)

Reply:
@mubaidr I appreciate your keenness to help, but I'm coaching @morena - which is what this repo was original for, so as much as all your feedback items are on point, please hold off. Thank you ❤️

Reply:
Did not know that. 👍